extensions/resteasy/runtime/src/main/java/io/quarkus/resteasy/runtime/NotFoundExceptionMapper.java

```diff
@@ -215,6 +214,13 @@ public Response toResponse(NotFoundException exception) {
                 .fromBoundResourceInvokers(bounded
                         .entrySet());
 
+        descriptions.sort(Comparator.comparing(description -> description.basePath));
+        for (ResourceDescription description : descriptions) {
+            description.calls.sort(
+                    Comparator.comparing((MethodDescription methodDesc) -> methodDesc.fullPath)
```

Comment:
So that's a good idea IMHO and this should get in.

Any chance you could get rid of the lambdas? We try to avoid them in runtime code as they have a memory cost.

Thanks!

Reply:
@gsmet
I changed it. Is it okay to create the Comparator inline like in the current version or do you prefer a static class?

Reply:
I would make it a static class.

Reply:
Done. Let me know if there's anything to improve.